src/tests/ftest/util/job_manager_utils.py

```diff
@@ -17,10 +17,62 @@
 from command_utils_base import FormattedParameter, EnvironmentVariables
 from exception_utils import CommandFailure, MPILoadError
 from env_modules import load_mpi
-from general_utils import pcmd, stop_processes, run_pcmd
+from general_utils import pcmd, stop_processes, run_pcmd, get_job_manager_class
 from write_host_file import write_host_file
 
 
+def get_job_manager(test, class_name=None, job=None, subprocess=None, mpi_type=None, timeout=None,
+                    namespace="/run/job_manager/*", class_name_default="Mpirun"):
+    """Get a JobManager object.
+
+    Create a JobManager class using either:
+        - the provided arguments
+        - the test yaml arguments (if no arguments are provided)
+        - default values (if no arguments are provided and there are no test yaml entries)
+
+    Args:
+        test (Test): avocado Test object
+        class_name (str, optional): JobManager class name. Defaults to None.
+        job (ExecutableCommand, optional): command object to manage. Defaults to None.
+        subprocess (bool, optional): whether the command is run as a subprocess. Defaults to False.
+        mpi_type (str, optional): MPI type to use with the Mpirun class only. Defaults to "openmpi".
+        timeout (int, optional): job manager timeout. Defaults to None.
+        namespace (str, optional): location of yaml parameters used to define unset inputs. Defaults
+            to "/run/job_manager/*".
+        class_name_default (str, optional): default class_name to use when. Defaults to "Mpirun".
+
+    Returns:
+        JobManager: a JobManager class, e.g. Orterun, Mpirun, Srun, etc.
+
+    """
+    job_manager = None
+    if class_name is None:
+        class_name = test.params.get("class_name", namespace, default=class_name_default)
```

Comment:
No, because the current behavior in test.py is to default to None so that a job manager is not created if one is not specified.   The default case is needed for all other instances.  See https://github.com/daos-stack/daos/pull/8060